tests/usecase/test_emit_metrics.py

```diff
@@ -15,6 +32,19 @@ def test_read_metrics_in_logs(self):
         self.assert_string_in_recent_logs("number_of_files")
         self.assert_string_in_recent_logs("critical_logs_count")
 
+    def test_free_apps_metrics(self):
```

Comment:
I dont think we need this method anymore, now that we have `TestFreeAppsMetricsEmitter`

Reply:
`TestFreeAppsMetricsEmitter` has more fine-grained unit tests as opposed to this integration test. I think it's actually better to have both.